front/components/assistant/conversation/RetrievalAction.tsx

```diff
@@ -15,6 +15,7 @@ import type {
 import { Transition } from "@headlessui/react";
 import { useState } from "react";
 
+import { makeLinkForRetrievedDocument } from "@app/components/actions/retrieval/utils";
```

Comment:
Will clean up this file in a follow up PR once all actions have been refactored.